datafusion/common/src/scalar.rs

```diff
@@ -3152,6 +3226,28 @@ impl fmt::Debug for ScalarValue {
             ScalarValue::FixedSizeList(_) => write!(f, "FixedSizeList({self})"),
             ScalarValue::List(_) => write!(f, "List({self})"),
             ScalarValue::LargeList(_) => write!(f, "LargeList({self})"),
+            ScalarValue::Struct(struct_arr) => {
```

Comment:
Likewise here -- perhaps we can use https://docs.rs/arrow/latest/arrow/util/display/fn.array_value_to_string.html  again